docs/content/contribute/styleguide.md

```diff
@@ -0,0 +1,324 @@
+---
+title: OSCAL Style Guide
+description: (with links) Contributors to OSCAL can save time by scanning this early.
+heading: OSCAL Style Guide
+weight: 10
+sidenav:
+    title: OSCAL Style Guide
+toc:
+    enabled: true
+    headingselectors: h2,h3,h4,h5
+    collapsedepth: 4
+custom_css:
+  - oscal-styleguide.css
+---
+
+## Goals, scope and audience
+
+This style guide is offered for authors and editors contributing to the NIST OSCAL project and related projects. Use it as a reference of first resort for questions of style or styling in contributions to OSCAL (treated broadly). At the foot are links to more extensive resources, which we recommend.
+
+The coverage here aims to be helpful and suggestive rather than normative. Feel free to create an [Issue on Github](https://github.com/usnistgov/OSCAL/issues/new?labels=User+Story%2C+enhancement%2C+Scope:%20Website&template=feature_request.md) to raise questions or points of discussion regarding either style (covered or not covered) or the Style Guide itself.
+
+The styling on our web sites is enabled and limited by the back-end architecture we use to produce and maintain them, the [US Web Design System](https://designsystem.digital.gov/), which we deliver via an [implementation](https://pages.nist.gov/hugo-uswds-docs/) of  [Hugo](https://gohugo.io/). See more below under [Hugo and Markdown](#hugo-and-markdown). Much of this guide assumes you are writing for a web site such as the [OSCAL web site](https://pages.nist.gov/OSCAL/) or for web-accessible resources such as `readme` documents in Github, and may be more or less applicable for other cases. Writing for OSCAL, we must rely on our readers to distinguish what advice is clearly not appropriate or useful to them, and the same rule applies to this page.
```

Comment:
The scope probably stretched a little however I discovered that: (a) the mechanics were not covered elsewhere, (b) they more or less had to be, among other reasons because they were sometimes (not always) implicated in matters of style; accordingly, a bare-bones treatment was judged to be in scope. Arguably a page on the mechanics could be carved out but TIASD (that is a separate discussion) raising questions of its own.

I suggest raising this as an Issue IFF it is found, shown or discovered to be a real issue for users.